six/modules/python/six.sicd/source/six_sicd.i

```diff
@@ -306,12 +308,13 @@ def readRegion(inputPathname, startRow, numRows, startCol, numCols, schemaPaths
     getWidebandRegion(inputPathname, schemaPaths, complexData, startRow, numRows, startCol, numCols, widebandBuffer)
 
     return widebandData, complexData
+
+
 def writeAsNITF(outFile, schemaPaths, complexData, image):
     writeNITF(outFile, schemaPaths, complexData,
         image.__array_interface__["data"][0])
 
 def readFromNITF(pathname, schemaPaths=VectorString()):
-    pathname = pathname + ".nitf"
```

Comment:
This seemed like a strange line to be in there... I assume that was in there by accident before and you removed it with intention here?